Pod/Classes/WPPHAssetDataSource.m

```diff
@@ -558,6 +558,11 @@ - (NSString *)fileExtension
     return [[[[PHAssetResource assetResourcesForAsset:self] firstObject] originalFilename] pathExtension];
 }
 
+- (nullable NSString *)UTTypeIdentifier
+{
+    return [self valueForKey:@"uniformTypeIdentifier"];
```

Comment:
I'm ok with using the private setter, but if this property changes name, will this method just return nil?

Reply:
Right! If the key where to change/be removed, we would get a awful crash: 
`... this class is not key value coding-compliant for the key ...`
We could:
- Take the risk.
- Let the scroll get slower with the safer method.
- Not show the badge on device media picker 😞 

I'll try to find another option for this, but so far, I haven't found a safe&fast way.

Reply:
you can do `valueForKey:@selector(uniformTypeIdentifier)` and if you rename it you'll get a compiler error

Reply:
(I think)

Reply:
Cool! Thank you!
Using that we can also check for `respondsToSelector`. That would save us from the runtime crashes in production.